webapp/packages/plugin-version-update-administration/src/PluginBootstrap.ts

```diff
@@ -23,7 +23,7 @@ export class PluginBootstrap extends Bootstrap {
     this.administrationItemService.create({
       name: 'version-update',
       type: AdministrationItemType.Administration,
-      order: 5,
+      order: 8,
```

Comment:
just remove `order`, so it will be set to `Number.MAX_SAFE_INTEGER` and always will be last element